bokeh/models/plots.py

```diff
@@ -56,12 +57,8 @@ def _select_helper(args, kwargs):
         selector = kwargs
     return selector
 
-class LayoutBox(Model):
-    ''' Represents an **on-canvas** layout.
-
-    '''
 
-class Plot(Component):
+class Plot(LayoutDOM):
```

Comment:
Say that this is a `DOM` layout doesn't seem necessary to the user. `Layout` would suffice.